### PR TITLE
Support Network Profiling

### PR DESCRIPTION
**Pull request by mrproliu**

### CODE REVIEW

File: configs/rover_configs.yaml [hunk 1]
```diff
@@ -112,5 +112,10 @@ profiling:
...
-      # the profiling stack dump period
-      dump_period: ${ROVER_PROFILING_TASK_ON_CPU_DUMP_PERIOD:9ms}
\ No newline at end of file
+      # The profiling stack dump period
+      dump_period: ${ROVER_PROFILING_TASK_ON_CPU_DUMP_PERIOD:9ms}
+    network:
+      # The interval of send metrics to the backend
+      report_interval: ${ROVER_PROFILING_TASK_NETWORK_TOPOLOGY_REPORT_INTERVAL:2s}
+      # The prefix of network profiling metrics name
+      meter_prefix: ${ROVER_PROFILING_TASK_NETWORK_TOPOLOGY_METER_PREFIX:rover_network_profiling}
```
[the remark this answers is not in the document]
Comment (mrproliu) [reply configs/rover_configs.yaml:121]: The name could be shorter. How about rename to "rover_network" or do you have any suggestions?

[the remark this answers is not in the document]
Comment (wu-sheng) [reply configs/rover_configs.yaml:121]: `rover_net_p_*` should be clear enough.

File: configs/rover_configs.yaml [hunk 2]
```diff
@@ -112,5 +112,10 @@ profiling:
...
-      # the profiling stack dump period
-      dump_period: ${ROVER_PROFILING_TASK_ON_CPU_DUMP_PERIOD:9ms}
\ No newline at end of file
+      # The profiling stack dump period
+      dump_period: ${ROVER_PROFILING_TASK_ON_CPU_DUMP_PERIOD:9ms}
+    network:
+      # The interval of send metrics to the backend
+      report_interval: ${ROVER_PROFILING_TASK_NETWORK_TOPOLOGY_REPORT_INTERVAL:2s}
+      # The prefix of network profiling metrics name
+      meter_prefix: ${ROVER_PROFILING_TASK_NETWORK_TOPOLOGY_METER_PREFIX:rover_net_p}
```
Comment (wu-sheng) [inline configs/rover_configs.yaml:121]: Is _ added automatically after prefix?

Comment (mrproliu) [reply configs/rover_configs.yaml:121]: Yes, it's added `_` automatically.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**